ui/tests/acceptance/task-detail-test.js

```diff
@@ -37,32 +38,37 @@ test('/allocation/:id/:task_name should name the task and list high-level task i
 
 test('breadcrumbs includes allocations and link to the allocation detail page', function(assert) {
   assert.equal(
-    find('[data-test-breadcrumb="allocations"]').textContent.trim(),
-    'Allocations',
-    'Allocations is the first breadcrumb'
+    find('[data-test-breadcrumb="jobs"]').textContent.trim(),
+    'Jobs',
+    'Jobs is in the breadcrumb trail'
   );
   assert.equal(
-    find('[data-test-breadcrumb="allocations"]').getAttribute('href'),
-    '#',
-    "Allocations breadcrumb doesn't link anywhere"
+    find('[data-test-breadcrumb="jobs.job"]').textContent.trim(),
+    job.name,
+    'Job name is in the breadcrumb trail'
   );
   assert.equal(
-    find('[data-test-breadcrumb="allocation"]').textContent.trim(),
+    find('[data-test-breadcrumb="jobs.job.task-group"]').textContent.trim(),
+    allocation.taskGroup, // not taskGroupName ?
```

Comment:
There is some unfortunate drift around Mirage models, application models, and the API. Some of this is just the way it has to be since the API and app are different, but there places (like this one) where I think it can be improved.

The explanation for this wart is the API returns `TaskGroup`, which gets aliased to `taskGroupName` [in the serializer](https://github.com/hashicorp/nomad/blob/master/ui/app/serializers/allocation.js#L9). It would be better to have the Mirage models better match the application code than the API.